atlp-rwanda/rca-barefoot-team4-bn
# #184638220 Integrate HoundCI with badge attached to readme

#### What does this PR do?

Integrate HoundCI with badge attached to readme

#### How should this be manually tested?

- Check that the badge code is correctly inserted into your README file. You should see the badge displayed in your README file, with a link to HoundCI.
- Push your changes to your repository and wait for HoundCI to complete the analysis. You should see the HoundCI status on your pull request or commit.

#### What are the relevant pivotal tracker/Trello stories?

#184638220
